setup.cfg

```
@@ -0,0 +1,21 @@
+[metadata]
+name = py-gene-fusions
+description = Package for gene fusions
+long_description = file:README.md
+long_description_content_type = text/markdown
+author = Wagner Lab, Nationwide Childrens Hospital
```

I would also add an email. Unsure of which one though
```suggestion
author = Wagner Lab, Nationwide Childrens Hospital
home-page = https://github.com/cancervariants/py-gene-fusions
license-file = LICENSE
```